jcl/src/java.base/share/classes/java/lang/invoke/MethodHandles.java

```diff
@@ -681,6 +717,32 @@ public MethodHandle findVirtual(Class<?> clazz, String methodName, MethodType ty
 			handle = SecurityFrameInjector.wrapHandleWithInjectedSecurityFrameIfRequired(this, handle);
 			return handle;
 		}
+
+		/*[IF Sidecar19-SE]*/
+		/**
+		 * Check if targetClass is in a package visible from the accessModule
+		 * @param accessModule module of the referring class
+		 * @param targetClass Class which the referring class is accessing
+		 * @param throwError Select the throwable thrown if the targetClass is not visible
+		 * @throws IllegalAccessError thrown if throwError is true
+		 * @throws IllegalAccessException thrown if throwError is false
+		 */
+		static void checkClassModuleVisibility(Module accessModule, Class<?> targetClass, boolean throwError) 
```

Comment:
Please put the throw clauses on the same line.

Reply:
Okay.